src/components/molecules/EntranceStepsInputFieldSet/EntranceStepsInputFieldSet.tsx

```diff
@@ -22,6 +29,7 @@ import { ButtonNG } from "components/atoms/ButtonNG";
 
 import "./EntranceStepsInputFieldSet.scss";
 
+let buttonsCopy: EntranceStepButtonConfig[] | undefined = [];
```

Comment:
Mutable shared state, `let` used instead of `const` on top of it. What's the point of this?

Reply:
removed 941192c